lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/MultiManagerProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+use function array_keys;
+
+final class MultiManagerProvider implements EntityManagerProvider
+{
+    /** @var EntityManagerInterface[] */
```

Comment:
To make Psalm happy:

```suggestion
    /** @var array<string, EntityManagerInterface> */
```